api/src/main/java/io/strimzi/api/kafka/model/Constants.java

```diff
@@ -11,5 +11,12 @@
     public static final String V1BETA1 = "v1beta1";
     public static final String V1ALPHA1 = "v1alpha1";
 
+    public static final String CRD_KIND = "CustomResourceDefinition";
+    public static final String STRIMZI_CATEGORY = "strimzi";
+
+    public static final String KUBERNETES_API_BUILDER = "io.fabric8.kubernetes.api.builder";
+    public static final String DONEABLE_PREFIX = "Doneable";
+    public static final String DONE = "done";
```

Comment:
I wonder about the wisdom of adding these. Let's remember that this is the `api` module, which we publish for people wanting to have a Java API for interacting with Strimzi. There's absolutely no benefit to then of having these constants, and they're unlikely to even know what they're for. But they have to be `public` to make the code work. TBH I could live with these ones remaining string literals to avoid adding public stuff which no one outside the api module needs to consume.